src/jit/compiler.cpp

```diff
@@ -4870,6 +4870,7 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
 #endif // defined(DEBUG)
 
     compFunctionTraceEnd(*methodCodePtr, *methodCodeSize, false);
+    JITDUMP("Method code size: %d", (unsigned)(*methodCodeSize));
 
```

Comment:
Yes, good point! It turns out that the dump of allocations starts with a newline, so I didn't notice it.